spark-api/lib/ie-contract.js

```diff
@@ -1,5 +1,5 @@
 import { ethers } from 'ethers'
-import { RPC_URL, GLIF_TOKEN } from '../spark-publish/ie-contract-config.js'
+import { RPC_URL, GLIF_TOKEN } from '../../spark-publish/ie-contract-config.js'
```

Comment:
what about moving `ie-contract-config.js` to a shared top level folder?

Reply:
That's a great idea!